curriculum/js1/4.js

```diff
@@ -7,7 +7,11 @@
  */
 
 const solution = (a, b) => {
-  return 0;
+    if(a <= 0){
+      return '';
+    }
+
+  return b + solution(a - 1 , b);
```

Comment:
please don't do this because it is inefficient. Pass the result in `solution = (a,b,result='') => {...`